Woof/Woof/View/LoginView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+
+/// A view that allows the user to select a role in the application.
+/// Based on the selected role, the user will be directed to the appropriate app flow.
```

Comment:
Are you sure that view knows something about the app navigation?

Reply:
I removed this line.